book.rb

```diff
@@ -0,0 +1,17 @@
+require './rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] As pointed out by the previous reviewer, it's important to set up a Setter for the `@rantals` instance as you did for `@title` and `@author`. This will allowed us to add, remove or modify a book **rentals**. 
**HIT:**
``` ruby
attr_accessor :title, :author, :rentals
```